_episodes/03-index-slice-subset.md

```diff
@@ -526,7 +526,7 @@ surveys_df[pd.isnull(surveys_df).any(axis=1)]
 {: .language-python}
 
 Note that the `weight` column of our DataFrame contains many `null` or `NaN`
-values. We will explore ways of dealing with this in Lesson 03.
+values. We will explore ways of dealing with this in [Lesson 04](https://datacarpentry.org/python-ecology-lesson/04-data-types-and-format/index.html).
```

Comment:
1. Here we're talking about the next *episode* (this whole repository is a single lesson, which is comprised of several episodes), so I suggest to tweak the wording and say something along the lines of `...explore ways of dealing with this in the next episode on [Data Types and Formats](...)`.
2. Let's use Carpentries' recommended way of linking to other episodes, which is described here: https://carpentries.github.io/lesson-example/04-formatting/index.html#locations-and-names